pizone/zone.py

```diff
@@ -88,6 +88,78 @@ def airflow_min(self) -> int:
         """Min allowed airflow for the zone as a percent"""
         return self._get_zone_state('MinAir')
 
+    async def set_airflow_min(self, value: int) -> None:
```

Comment:
This command is only available in the v2 API. Please put in a check to ensure that the controller is in v2 mode and throw an exception if not. There's an attribute set on the controller.

Reply:
Are you sure? It's in the v1 API documentation and I have a v1 interface and can issue this command without issue.

Reply:
Oh, I thought I'd completely implemented the v1 api. It was a long time ago though, maybe I'm wrong. In that case yeh that's fine and your refactoring of the function makes sense too.